packages/cli/lib/services/dryrun.service.ts

```diff
@@ -168,14 +168,11 @@ export class DryRunService {
             lastSyncDate = new Date(suppliedLastSyncDate);
         }
 
-        let type = 'sync';
-
+        let type: ScriptFileType = 'syncs';
         if (syncInfo?.type === 'action') {
-            type = 'action';
-        }
-
-        if (isPostConnectionScript) {
-            type = 'post-connection-script';
+            type = 'actions';
+        } else if (isPostConnectionScript) {
+            type = 'post-connection-scripts';
         }
```

Comment:
type is being pluralized. Does that have any impact on already existing `nango-intergrations` folders?

Reply:
we used to add an `s` dynamically to this variable in another function so it's actually simplifying the code